src/components/GridItems/GridItems.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import CoffeeItem from "../CoffeeItem/CoffeeItem";
+import CoffeItems from "../../data/coffeeItems.js";
```

Comment:
Should a camel case 
Add an entry point index.js to the data folder 
```suggestion
import CoffeItems from "../../data/coffeeItems.js";
```